scripts/processing/05-run_ica.py

```diff
@@ -32,6 +33,10 @@ def run_ica(subject_id, tsss=None):
         else:
             run_fname = op.join(data_path, 'run_%02d_filt_sss_highpass-%sHz'
                                 '_raw.fif' % (run, 1))
+        if not op.exists(run_fname):
+            warn('Could not find file %s. '
+                 'Skipping run %s for subject %s.' % (run_fname, run, subject))
+            continue
```

Comment:
how did you hit this?

Reply:
When running the ICA script. Same as in the filtering script, no?

Reply:
Maybe we don't need them in either place, it might be better to raise an error instead. It seems indicative that some previous step was skipped

Reply:
I thought some files on OpenfMRI were corrupted? According to [this comment](https://github.com/mne-tools/mne-biomag-group-demo/blob/master/scripts/processing/04-python_filtering.py#L41) (probably by Jaakko)

Reply:
I don't remember hitting the problem. Maybe it as an old version of the analysis. I'll try running that script without the `try`

Reply:
okay let me know. Otherwise, I may have to download the dataset again

Reply:
it just ran without error for me with no `try` clause, so let's remove it